simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/SetVariableMonitoringRequestHandler.java

```diff
@@ -0,0 +1,73 @@
+package com.evbox.everon.ocpp.simulator.station.handlers.ocpp;
+
+import com.evbox.everon.ocpp.common.CiString;
+import com.evbox.everon.ocpp.simulator.station.StationMessageSender;
+import com.evbox.everon.ocpp.simulator.station.component.StationComponent;
+import com.evbox.everon.ocpp.simulator.station.component.StationComponentsHolder;
+import com.evbox.everon.ocpp.v20.message.centralserver.SetMonitoringDatum;
+import com.evbox.everon.ocpp.v20.message.centralserver.SetMonitoringResult;
+import com.evbox.everon.ocpp.v20.message.centralserver.SetVariableMonitoringRequest;
+import com.evbox.everon.ocpp.v20.message.centralserver.SetVariableMonitoringResponse;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Handler for {@link SetVariableMonitoringRequest} request.
+ */
+@Slf4j
+public class SetVariableMonitoringRequestHandler implements OcppRequestHandler<SetVariableMonitoringRequest> {
+
+    private StationComponentsHolder stationComponentsHolder;
+    private StationMessageSender stationMessageSender;
+
+    public SetVariableMonitoringRequestHandler(StationComponentsHolder stationComponentsHolder, StationMessageSender stationMessageSender) {
+        this.stationComponentsHolder = stationComponentsHolder;
+        this.stationMessageSender = stationMessageSender;
+    }
+
+    /**
+     * Handle {@link SetVariableMonitoringRequest} request.
+     *
+     * @param callId identity of the message
+     * @param request incoming request from the server
+     */
+    @Override
+    public void handle(String callId, SetVariableMonitoringRequest request) {
+        List<SetMonitoringResult> results = new ArrayList<>();
+
+        for (SetMonitoringDatum data : request.getSetMonitoringData()) {
+            SetMonitoringResult monitoringResult = buildResponse(data);
+
+            String componentName = data.getComponent().getName().toString();
+            Optional<StationComponent> component = stationComponentsHolder.getComponent(new CiString.CiString50(componentName));
```

Comment:
Such usage of Optional seems to be awkward a bit.
All execution paths return some object to be added into results list so `component.map().orElseGet()` seems to be natural